core/arch/asm_defines.asm

```diff
@@ -61,6 +61,11 @@
 # define AARCHXX
 #endif
 
+//NOCHECK
+#define X86
```

Comment:
This is WIP code, so I guess this define is hard-wired because you are assuming an X86 host and just want to get things going right?

Reply:
Yes, hadn't quite cleaned everything up when you requested a look.  This will use the DR_HOST_* defines (which didn't exist when this hack was first put in).